provisioning/local/aws.tf

```diff
@@ -0,0 +1,88 @@
+provider "aws" {
+  region  = local.region
+  profile = var.aws_profile
+}
+
+locals {
+  region = "eu-central-1"
+}
+
+variable "aws_profile" {
+  type = string
+  validation {
+    condition     = length(var.aws_profile) > 0
+    error_message = "The \"aws_profile\" must be non-empty string."
+  }
+}
+
+resource "aws_iam_user" "api_error_control_iam_user" {
```

Comment:
Ten `_iam_user` suffix v `api_error_control_iam_user` je vice/mene zbytecnej, protoze to je IAM user resource. Asi neco jako piseme `Interface` suffix na konec nazvu interface :slightly_smiling_face: 

To samy plati i pro pojmenovani ostatnich resources niz.